components/Greeting/Greeting.module.css

```diff
@@ -0,0 +1,22 @@
+.grettingContainer {
+  width: 350px;
+  height: 70px;
```

Comment:
Is it required to set absolute values here?
This will lead to issues if you change the content.

E.g. try to set a longer name in your Greeting component. If it is too long, the content will overflow.

Thus, it's not recommended to set absolute width and height.

This issue happens in other components too.